packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -225,9 +225,17 @@ export default {
                 basket: {
                     id: data.BasketId,
                     total: data.BasketSummary.BasketTotals.Total
-                }
+                },
+                ageRestricted: data.BasketSummary.Prompts.Restrictions.some(restriction => restriction.Type === 'Alcohol')
             };
 
+            // This logic is temporary while a new endpoint is built for fulfilment status.
+            // We can't call GET checkout as a guest so we're now having to use the basket response for age restrictions until then
+            if (basketDetails.ageRestricted && (tenant === 'au' || tenant === 'nz')) {
```

Comment:
Is this how we do feature toggling in Fozzie components?

Reply:
I would actually like to know how to do feature toggling for another feature I've built (split notes) but this isn't it. In this case, I'm just checking if it's AU/NZ because we're using the response from basket to determine if the age verification should be shown while we await a new checkout endpoint. The checkout endpoint will do the tenant checking for us, whereas I believe Basket will return the age restricted prompt regardless of country (although i may be wrong with that)

Reply:
Yeah sure, I think in some of the other modules the toggling is done via a config file where you set variables per tenant, pass in the current tenant from the consuming app, and read the relevant config out and use that throughout the components - that way each tenants config is in a self-contained file.

Another approach could be similar to how menu does feature toggling where we have a JS feature toggle package that accepts the current env, tenant etc, then reads in some JSON and can tell if a feature should be enabled or not.

As the first approach is already in use I'd suggest going with that one, maybe in a future PR if it's a lot of work?

Reply:
Sure, give me a shout any time (apart from next week as I'm on leave!).